generator/internal/scaffold_generator.cc

```diff
@@ -326,37 +325,6 @@ void GenerateScaffold(
   GenerateQuickstartWorkspace(os, vars, contents);
 }
 
-void GenerateCmakeConfigIn(
```

Comment:
There is a declaration for this in the header.

Reply:
D'oh. Removed.